telescope_config.yaml

```diff
@@ -4,26 +4,39 @@ metrics:
     global:
         scrape_interval: 15s
         external_labels:
-            project_id: hyperbridge-nexus-bootnode
-            project_name: hyperbridge-nexus
+            project_id: adedayo123
+            project_name: 0hxNEtwork
         remote_write:
-            - url: https://hyperbridge-prometheus.blockops.network/api/v1/write
+            - url: http://telescope.blockops.network
               basic_auth:
-                password: hyperbridge
-                username: hyperbridge
+                password: pass
+                username: user
     wal_directory: /tmp/wal
     configs:
-        - name: hyperbridge-nexushyperbridge_metrics
+        - name: 0hxnetworkethereum_metrics
           host_filter: false
           scrape_configs:
-            - job_name: hyperbridge-nexus_hyperbridge_relaychain_job_0
+            - job_name: 0hxnetwork_ethereum_execution_job_0
               static_configs:
                 - targets:
-                    - localhost:9615
-            - job_name: hyperbridge-nexus_hyperbridge_parachain_job_1
+                    - localhost:6060
+            - job_name: 0hxnetwork_ethereum_consensus_job_1
               static_configs:
                 - targets:
-                    - localhost:9616
+                    - localhost:8008
+logs:
```

Comment:
looks the [log-sink url](https://github.com/blockopsnetwork/telescope/pull/19/files#diff-61358711e980ccf505246fd3915f97cbd3a380e9b66f6fa5aad46749968c5ca3R389) is required when `enable-logs` is set to true.. or is it optional?

Reply:
Yes, thats true, fow now the user is required to provide the log sink url